gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/ProjectController.groovy

```diff
@@ -44,23 +44,13 @@ class ProjectController {
 
   @RequestMapping(value = "/{id:.+}", method = RequestMethod.GET)
   Map get(@PathVariable("id") String projectId) {
-    def result = projectService.get(projectId)
-    if (!result) {
-      log.warn("Project not found (projectId: ${projectId}")
-      throw new ProjectNotFoundException("Project not found (projectId: ${projectId})")
-    }
-    result
+    return projectService.get(projectId)
```

Comment:
The previous bits were bogus anyways ... you'll never get a `null` value back (rather a RetrofitError).